### ISSUE TITLE
Possible simplification in T5 docs

### ISSUE THREAD

**Issue by xhlulu** [huggingface/transformers#14731]
I noticed the following nested list comprehension being recommended in the [T5 docs:](https://huggingface.co/docs/transformers/model_doc/t5)

```python
# replace padding token id's of the labels by -100
labels = [
           [(label if label != tokenizer.pad_token_id else -100) for label in labels_example] for labels_example in labels
] 
labels = torch.tensor(labels)
```

I believe it's possible to simplify above to:
```python
labels = torch.tensor(labels)
labels[labels == tokenizer.pad_token_id] = -100
```

I would be happy to make a PR if the change is deemed relevant, though I'm not sure where the docs are located so please let me know if needed :)

**Comment by NielsRogge** [comment]
Hey,

that's indeed a way more readable way of replacing padding tokens by -100. 

The docs of T5 is located [here](https://github.com/huggingface/transformers/blob/master/docs/source/model_doc/t5.rst). Note that we are migrating all docs from .rst files to Markdown. But for now, you can add just update the code example.

**Comment by xhlulu** [comment]
Thanks for sharing! I made a PR
